quetz/main.py

```diff
@@ -1382,7 +1383,9 @@ async def post_upload(
             status_code=status.HTTP_406_NOT_ACCEPTABLE, detail="Wrong SHA256 checksum"
         )
 
-    user_id = auth.assert_user()
+    # get the id of the owner, in case auth was done through an API key
+    owner_id = auth.assert_owner()
```

Comment:
Probably same comment here to keep it consistent: `user_id` instead of `owner_id` for consistency with `dao.create_package`